### ISSUE TITLE
[Effect Suggestion] Explosive Sniper Shots

### ISSUE THREAD

**Issue by ThatDeathbat** [gta-chaos-mod/ChaosModV#452]
**Name**
   -  Sniper Rifle Obliteration 

**Duration**
   -  Long

**Description**
Everybody likes explosions, don’t they? 
Now every shot fired from a **_SNIPER RIFLE_** causes a big explosion at the location the bullet hits!  Its great for quick crowd control, but not so great for if being sneaky is the objective.

Kinda like a railgun, but a bigger explosion and a longer range!

**Comment by ThatDeathbat** [comment]
Sweet idea, right?!

**Comment by Last0xygen** [comment]
Already there with „explosive combat“, which gives explosive ammo for all weapons 👍

**Comment by ThatDeathbat** [comment]
This is solely for the sniper rifle and has a HUGE explosion radius.

**Comment by Last0xygen** [comment]
Then it would be too situational. The player needs to have a sniper AND needs to use it to even see any effect

**Comment by ThatDeathbat** [comment]
True, but most players see the effect at the left and will decide if they want to utilize it.

**Comment by ThatDeathbat** [comment]
> Then it would be too situational imho. The player needs to have a sniper AND needs to use it to even see any effect

Most players have a sniper not long into story mode, unless they drop it.

**Comment by anto4440** [comment]
Just too specific

**Comment by SeanMirrsen** [comment]
You could make it better by giving it to the player, and forcing them to use it. Call the effect something different, like Davy Crockett Snipers. It'll be a somewhat less chaotic version of giving everyone an RPG, it'll be just the player with the sniper, and any snipers currently about (which there are a few on some missions).
